superset/viz.py

```diff
@@ -1354,7 +1360,7 @@ def get_df(self, query_obj=None):
         fd = self.form_data
 
         row = df.groupby(self.groupby).sum()[self.metrics[0]].copy()
-        row.sort_values(ascending=False, inplace=True)
+        # row.sort_values(ascending=False, inplace=True)
```

Comment:
@xrmx 
add this line:(Disorder)
![a](https://cloud.githubusercontent.com/assets/19641641/20922593/b082c3ae-bbe3-11e6-8e81-f7348c775a13.png)

remove this line:(Order)
![b](https://cloud.githubusercontent.com/assets/19641641/20922719/11bde00e-bbe4-11e6-9277-bef7265025ce.png)